mlflow/tensorflow/__init__.py

```diff
@@ -900,6 +984,94 @@ def _log_early_stop_callback_metrics(callback, history, metrics_logger):
         if metric_key is not None:
             metrics_logger.record_metrics(restored_metrics, stopped_epoch + 1)
 
+    def _log_keras_model(history, args):
+        def _get_input_data_slice():
```

Comment:
Similar to https://github.com/mlflow/mlflow/pull/5510/files#r851523774, can we move this to `_autolog.py`?